app/views/posts/index.html.erb

```diff
@@ -11,6 +11,9 @@
           <td><%= post.title %></td>
           <td><%= post.text %></td>
           <td><%= post.user.name %></td>
+          <td><%= link_to 'Delete', post_path(post),
```

Comment:
Should anyone be able to delete a post?

Reply:
Now user can only delete own posts

Reply:
It's good that you added that `if` statement to the controller action. But personally, I feel that it is not good UX to show the user a button for deleting a post when they cant delete it.

Reply:
After the work we had building that if statement!! It hurts but you're totally right! It's fixed now! Only the post owner can see the Delete button!